tf2onnx/function/range.py

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+
+"""
+tf2onnx.tf2onnx - range op conversion
+"""
+import numpy as np
+from onnx import helper
+from onnx.onnx_pb import TensorProto
+from tf2onnx import utils
+
+# pylint: disable=useless-return,broad-except,logging-not-lazy,unused-argument,missing-docstring
+def make_range_const(ctx,
```

Comment:
remove these pylint rule unless necessary. i am pretty sure there is no broad-except or useless-return

Reply:
so for the other files.